build_tools/cmake/build_and_test_asan.sh

```diff
@@ -122,46 +122,19 @@ label_exclude_regex="($(IFS="|" ; echo "${label_exclude_args[*]?}"))"
 
 vulkan_label_regex='^driver=vulkan$'
 
-# These tests currently have asan failures
-declare -a excluded_tests=(
-  # TODO(#5716): Fix flaky ASan crash in these tests
-  "iree/tests/e2e/models/collatz.mlir.test"
-  "iree/tests/e2e/models/edge_detection.mlir.test"
-  "iree/tests/e2e/models/fragment_000.mlir.test"
-  "iree/tests/e2e/models/fullyconnected.mlir.test"
-  "iree/tests/e2e/models/mnist_fake_weights.mlir.test"
-  "iree/tests/e2e/models/resnet50_fake_weights.mlir.test"
-  "iree/tests/e2e/models/unidirectional_lstm.mlir.test"
-  "iree/tests/e2e/regression/globals.mlir.test"
-  # TODO(#5715): Fix these
-  "iree/samples/simple_embedding/simple_embedding_vulkan_test"
-  "iree/tools/test/iree-benchmark-module.mlir.test"
-  "iree/tools/test/iree-run-module.mlir.test"
-  "iree/tools/test/multiple_exported_functions.mlir.test"
-)
-
-# Prefix with `^` anchor
-excluded_tests=( "${excluded_tests[@]/#/^}" )
-# Suffix with `$` anchor
-excluded_tests=( "${excluded_tests[@]/%/$}" )
-
-# Join on `|` and wrap in parens
-excluded_tests_regex="($(IFS="|" ; echo "${excluded_tests[*]?}"))"
-
 cd ${BUILD_DIR?}
 
-echo "********** Running main project ctests on non-Vulkan backends ***********"
+echo "*** Running main project ctests that do not use the Vulkan driver *******"
 ctest \
   --timeout 900 \
   --output-on-failure \
   --no-tests=error \
-  --label-exclude "${label_exclude_regex}|${vulkan_label_regex}" \
-  --exclude-regex "${excluded_tests_regex?}"
+  --label-exclude "${label_exclude_regex}|${vulkan_label_regex}"
 
 echo "******************** llvm-external-projects tests ***********************"
 cmake --build . --target check-iree-dialects -- -k 0
 
-echo "**************** Running main project ctests on Vulkan ******************"
+echo "*** Running main project ctests that may use the Vulkan driver  *********"
```

Comment:
I think this is *do* use the vulkan driver. It's limited to specifically vulkan tests

```suggestion
echo "*** Running main project ctests use the Vulkan driver  *********"
```

Reply:
Maybe also include that LSAN is disabled?

```suggestion
echo "*** Running main project ctests using the Vulkan driver without LSAN  *********"
```

Reply:
done.